<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>652</width>
    <height>505</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="b_encrypt_btn">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>160</y>
      <width>101</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Basic Encryption</string>
    </property>
   </widget>
   <widget class="QPushButton" name="b_decrypt_btn">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>190</y>
      <width>101</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Basic Decrypt</string>
    </property>
   </widget>
   <widget class="QPushButton" name="psw_encrypt_btn">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>160</y>
      <width>131</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Password Encryption</string>
    </property>
   </widget>
   <widget class="QPushButton" name="psw_decrypt_btn">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>190</y>
      <width>131</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Password Decrypt</string>
    </property>
   </widget>
   <widget class="QPushButton" name="file_btn">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>310</y>
      <width>141</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>Open File</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>380</y>
      <width>111</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Select File</string>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="psw_edit">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>310</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
